containers/java-postgres/test-project/src/test/java/com/mycompany/app/AppTest.java

```diff
@@ -0,0 +1,44 @@
+/*--------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+package com.mycompany.app;
+
+import org.junit.Test;
+
+import junit.framework.AssertionFailedError;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+
+public class AppTest
+{
+    App newApp = new App();
+    public AppTest() {
+    }
+
+    @Test
+    public void testApp()
+    {
+        assertTrue( true );
+    }
+
+    @Test
+    public void testMore()
```

Comment:
Same comment as testApp()

Reply:
that was an existing @Test taken from the `java` folder that we left in place.

Reply:
+1